lib/puppet/type/yumrepo.rb

```diff
@@ -423,6 +421,13 @@
     desc "Password to use with the username for basic authentication.
       #{ABSENT_DOC}"
     newvalues(%r{.*}, :absent)
-    sensitive true
+  end
+
+  private
+
+  def set_sensitive_parameters(sensitive_parameters) # rubocop:disable Style/AccessorMethodName
```

Comment:
Do we need to reenable the cop at the end of the method too? Or does that just disable it for the one method?

Reply:
No, apparently we do not
```
[0] Melissa@bubba:~/puppetlabs-yumrepo_core:(master)$ be rubocop -D
Inspecting 10 files
.........C

Offenses:

lib/puppet/type/yumrepo.rb:434:7: C: Style/AccessorMethodName: Do not prefix writer method names with set_.
  def set_other_thing(sensitive_parameters)
      ^^^^^^^^^^^^^^^

10 files inspected, 1 offense detected
```
my test file:
```
426   private
427
428   def set_sensitive_parameters(sensitive_parameters) # rubocop:disable Style/AccessorMethodName
429     parameter(:password).sensitive = true if parameter(:password)
430     parameter(:proxy_password).sensitive = true if parameter(:proxy_password)
431     super(sensitive_parameters)
432   end
433
434   def set_other_thing(sensitive_parameters)
435     # does nothing
436     parameter(:password).sensitive = true if parameter(:password)
437     parameter(:proxy_password).sensitive = true if parameter(:proxy_password)
438     super(sensitive_parameters)
439   end
440 end
```